app/page/template/modals.htm

```diff
@@ -267,7 +267,7 @@
     </div>
   </div>
 
-  <div class="modal modal-connected-device" data-uie-name="modal-conntected-device">
+  <div class="modal modal-connected-device" data-uie-name="modal-connected-device">
```

Comment:
@Yserz & @mythsunwind Can you please update our tests to use this new `data-uie-name`?